test/parallel/test-stdin-require.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const execSync = require('child_process').execSync;
+const fs = require('fs');
+const path = require('path');
+
+if (common.isWindows) {
+  common.skip('Skipping test on windows');
```

Comment:
I'll spend a bit trying to reproduce on windows and if I can't, I'll update the message